packages/flutter/lib/src/material/drawer.dart

```diff
@@ -228,6 +227,14 @@ class DrawerController extends StatefulWidget {
   /// By default, the color used is [Colors.black54]
   final Color scrimColor;
 
+  /// The width of the area within which a horizontal swipe will open the
+  /// drawer.
+  ///
+  /// By default, the value used is 20.0 added to the value of
+  /// `MediaQuery.of(context).padding`. This ensures that the drag area for
```

Comment:
You're adding 20 to the padding edge that corresponds to [alignment], not to the padding itself.